README.md

```diff
@@ -37,22 +37,24 @@ Or add it as a script to your `package.json`
-| NPM cache             | `npm cache verify`               | Yes                      | Yes        | true     | --keep-node-modules    |
-| NPM Install           | `npm ci`                         | Yes                      | Yes        | true     | --keep-node-modules    |
-| iOS build folder      | `rm -rf ios/build`               | Yes                      | Yes        | false    | --remove-iOS-build     |
-| iOS pods folder       | `rm -rf ios/pods`                | Yes                      | Yes        | false    | --remove-iOS-pods      |
-| Android build folder  | `rm -rf android/build`           | Yes                      | Yes        | false    | --remove-android-build |
-| Android clean project | `(cd android && ./gradlew clean)`| Yes                      | Yes        | false    | --clean-android-project|
-| Brew package          | `brew update && brew upgrade`    | No                       | Yes        | true     | --keep-brew            |
-| Pod packages          | `pod update`                     | No                       | Yes        | true     | --keep-pods            |
+| State Type            | Command                           | In `clean-project-auto`? | Optional? | Default? | Option Flag                |
+| --------------------- | --------------------------------- | ------------------------ | --------- | -------- | -------------------------- |
+| React-native cache    | `rm -rf $TMPDIR/react-*`          | Yes                      | No        | true     |                            |
+| Metro bundler cache   | `rm -rf $TMPDIR/metro-*`          | Yes                      | No        | true     |                            |
+| Watchman cache        | `watchman watch-del-all`          | Yes                      | No        | true     |                            |
+| NPM modules           | `rm -rf node_modules`             | Yes                      | Yes       | true     | --keep-node_modules        |
+| Yarn cache            | `yarn cache clean`                | Yes                      | Yes       | true     | --keep-node-modules        |
+| Yarn packages         | `yarn install`                    | No                       | Yes       | true     | --keep-node-modules        |
+| NPM cache             | `npm cache verify`                | Yes                      | Yes       | true     | --keep-node-modules        |
+| NPM Install           | `npm ci`                          | Yes                      | Yes       | true     | --keep-node-modules        |
+| iOS build folder      | `rm -rf ios/build`                | Yes                      | Yes       | false    | --remove-iOS-build         |
+| iOS pods folder       | `rm -rf ios/Pods`                 | Yes                      | Yes       | false    | --remove-iOS-pods          |
+| system pods cache     | `pod cache clear --all`           | Yes                      | Yes       | false    | --remove-system-pods-cache |
```

Comment:
I suggest keeping the `iOS` naming convention here. So instead of `system pods cache`, I'd use `iOS system pods cache`, favouring verbosity.

Reply:
sure thing @pmadruga - I just swapped names so that 'iOS' is in there in all places and re-pushed